app/src/main/java/org/fossasia/openevent/app/common/data/repository/AttendeeRepository.java

```diff
@@ -117,10 +118,12 @@ public Completable scheduleToggle(Attendee attendee) {
 
                 return eventService.patchAttendee(attendee.getId(), attendee);
             })
-            .doOnNext(attendee -> {
+            .flatMap(attendee -> {
                 databaseRepository
                     .update(Attendee.class, attendee)
                     .subscribe();
+
+                return diskObservable;
```

Comment:
this will remove onError action from `eventService.patchAttendee` to `diskObservable`